technologies/markup.md

```diff
@@ -0,0 +1,48 @@
+# Markup
+
+## Background
+
+HTML up to 3.2 was written with SGML syntax rules.  HTML 4 could be written with either SGML syntax (HTML 4.01) or XML (XHTML 1.0).  [HTML 5 breaks with SGML and has two serializations](https://www.w3.org/blog/2008/01/html5-is-html-and-xml/), a new one called "html" (which looks like SGML but is not an application of SGML) and again XML (XHTML).
```

Comment:
This is really great as a whole. Thanks! I am wondering if we can make it more concise. We want the playbook as a whole to be a compelling read, therefore I'm wondering if this background is needed? 

If you feel it is, what are you thoughts on adding it to the bottom as a 'Further Reading' section?